website/content/docs/secrets/aws.mdx

```diff
@@ -474,6 +474,40 @@ have a time-to-live of 1 hour and automatically refresh when they expire.
 Please see the [API documentation](/vault/api-docs/secret/aws#configure-root-credentials)
 for more details on the fields associated with plugin WIF.
 
+## Cross-account static role management
+
+<EnterpriseAlert product="vault" />
```

Comment:
```suggestion
@include 'alerts/enterprise-only.mdx'
```

We have dedicated partials for Enterprise banners to keep language consistent and differentiate between enterprise features that are only available in self managed enterprise and features that are available in enterprise *or* HVD.  If the feature is also available in HVD, use the 'enterprise-and-hcp.mdx' partial instead.